src/libs/feature-previews-config.ts

```diff
@@ -112,6 +124,7 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
     feedbackUrl: `mailto:dsp-core-services@broadinstitute.org?subject=${encodeURIComponent(
       'Feedback on Improved Spend Reports'
     )}`,
+    lastUpdated: '11/19/2024',
```

Comment:
Are we going to know that these need to be updated if these things change? It seems like the sort of thing that will quickly be inaccurate

Reply:
@admull can you take this one?